aws/extensions/computetask_awswin_ecs/extension.ftl

```diff
@@ -0,0 +1,227 @@
+[#ftl]
+
+[@addExtension
+    id="computetask_awswin_ecs"
+    aliases=[
+        "_computetask_awswin_ecs"
+    ]
+    description=[
+        "Setup the ecs agent and docker config for aws win instances"
+    ]
+    supportedTypes=[
+        ECS_COMPONENT_TYPE
+    ]
+    scopes=[
+        COMPUTETASK_EXTENSION_SCOPE
+    ]
+/]
+
+[#macro shared_extension_computetask_awswin_ecs_deployment_computetask occurrence ]
+
+    [#local solution = occurrence.Configuration.Solution]
+    [#local resources = occurrence.State.Resources]
+
+    [#local ecsId               = resources["cluster"].Id ]
+    [#local dockerUsers         = solution.DockerUsers ]
+    [#local dockerVolumeDrivers = solution.VolumeDrivers ]
+    [#local defaultLogDriver    = solution.LogDriver ]
+
+    [#local dockerUsersEnv = "" ]
+
+    [#local commands = {}]
+    [#local services = {}]
+
+    [#local solution = occurrence.Configuration.Solution ]
+    [#local operatingSystem = solution.ComputeInstance.OperatingSystem]
+
+    [#local userInit = {}]
+    [#if dockerUsers?has_content ]
+        [#list dockerUsers as userName,details ]
+
+            [#local userInit = mergeObjects(
+                                userInit,
+                                {
+                                    userName : {
+                                        "groups" : [ "docker" ],
+                                        "uid" : details.UID
+                                    }
+                                })]
+        [/#list]
+    [/#if]
```

Comment:
Might need to review this and see what the windows config is for user provisioning.

Reply:
Sorry yes, ecs is currently junk and a placeholder for me